packages/flutter_nekoton_bridge/example/integration_test/test_helpers.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:io';
+
+String _envStringSkipFlakyIOS = 'FLUTTER_TEST_SKIP_FLAKY_IOS';
+String _envStringSkipFlakyAndroid = 'FLUTTER_TEST_SKIP_FLAKY_ANDROID';
+
+bool getBooleanEnv(String key) {
+  final value = Platform.environment[key];
+  switch (value?.toLowerCase() ?? false) {
```

Comment:
If `value` is string, then `?? 'false'` must be, isn't it?

Reply:
Whoa! Thank you!

Reply:
fixed.